core/arch/aarch64/codec.c

```diff
@@ -2789,14 +2789,14 @@ decode_ldstex(dcontext_t *dcontext, byte *pc_, byte *orig_pc_, instr_t *instr_ld
 }
 
 byte *
-decode_common(dcontext_t *dcontext, byte *pc, byte *orig_pc, instr_t *instr)
+decode_common_with_ldstex(dcontext_t *dcontext, byte *pc, byte *orig_pc, instr_t *instr)
```

Comment:
There are strong arguments for not putting this bundling inside the aarch64 decoder:

-  Modularity: the decoder should stick to its primary purpose, decoding individual instructions.  We should be able to take a 3rd-party decoder and pug it in here.  Bundling is a higher-level concept related to blocks, not to decoding.

- Generality: we might want to do the same thing for aarch32.  This bundling doesn't seem to be an aarch64-specific thing.

- Consistency: existing bundling for Level 0 instructions is a higher-level action taken when building a basic block.  It does not live inside the decoder.

What are the reasons to put this inside the aarch64 decoder?